public/assets/elfrun.js

```diff
@@ -42,17 +49,23 @@ function elfRunNextInst() {
                         stopCount = 200000;
                     } else {
                         stopCount = 10000;
+                        if (command != null && command[0] === '\r') {
+                            console.log("returning nextinstr");
+                        }
                         return false;
                     }
                 }
             }
         }
-
-
+        if (command != null && command[0] === 'l') {
+            console.log(RISCV.pc);
+        }
         // set last PC value for comparison
         //RISCV.oldpc = RISCV.pc;
-        
         instVal = RISCV.load_inst_from_mem(RISCV.pc);
+        if (command != null && command[0] === '\r') {
```

Comment:
same here.

also since you do the check a lot, what is the behavior if command is null? should the function throw an error? maybe we can check `if (!command)` at the beginning and then the rest of the function could assume the command is non-null